packages/logs/src/domain/trackConsoleLogs.spec.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-console */
+
+import { trackConsoleLogs } from './trackConsoleLogs'
+;(['log', 'info', 'warn', 'debug'] as const).forEach((status) => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏spacing seems weird to me, we could maybe check if it can be enforced
```suggestion
/* eslint-disable no-console */
import { trackConsoleLogs } from './trackConsoleLogs'

;(['log', 'info', 'warn', 'debug'] as const).forEach((status) => {
```

Reply:
The space between the import and the first line does not respect our prettier rules, which is weird. I'll look into it

Reply:
I submitted the prettier issue https://github.com/prettier/prettier/issues/12211